tests/rados/test_osd_bluestore.py

```diff
@@ -0,0 +1,79 @@
+"""
+This module tests the blue store related feature-
+ 1. Testing blue store v3
+"""
+
+import random
+import time
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from utility.log import Log
+from utility.utils import method_should_succeed
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_obj = RadosOrchestrator(node=cephadm)
+
+    log.info("Running blue store V3 test case")
+    if config.get("create_pools"):
+        pools = config.get("create_pools")
+        for each_pool in pools:
+            pool_details = each_pool["create_pool"]
+            pool_name = pool_details.get("pool_name")
+            pg_num = pool_details.get("pg_num")
+            method_should_succeed(
+                rados_obj.create_pool, pool_name=pool_name, pg_num=pg_num
+            )
+            rados_obj.bench_write(pool_name=pool_name, rados_write_duration=300)
+
+    # Get the osd clients
+    clients = ceph_cluster.get_ceph_objects("osd")
+
+    # get the OSD list as a dictionary.Node name as key and osd numbers as values
+    rados_list = rados_obj.get_osd_list()
+
+    # Randomly picking the osd node from the list and in that osd node again picking random osd.
+    # kill the osd and checking the osd process will up and run automatically.
+    # Executing the test three times.
+
+    for _ in range(3):
+        # Randomly pick the osd object.
+        host_object = random.choice(clients)
```

Comment:
after addressing the above comment, Instead of `clients` we could use hosts from `get_osd_list`.